saiph/reduction/famd_sparse.py

```diff
@@ -18,6 +18,9 @@ def fit(
     df: pd.DataFrame,
     nf: Optional[int] = None,
     col_weights: Optional[NDArray[np.float_]] = None,
+    seed: Optional[
+        int
+    ] = None,  # Even if not used, a fit function accepts a seed argument (typecheck)
```

Comment:
```suggestion
    ] = None,
```

This is good as a PR comment, but I don't think it's needed as a comment in the code.